parents/integrationtest/pom.xml

```diff
@@ -140,33 +140,56 @@
                     </executions>
                 </plugin>
                 <plugin>
-                    <groupId>com.github.alexcojocaru</groupId>
-                    <artifactId>elasticsearch-maven-plugin</artifactId>
-                    <version>${version.com.github.alexcojocaru.elasticsearch.plugin}</version>
+                    <groupId>io.fabric8</groupId>
+                    <artifactId>docker-maven-plugin</artifactId>
+                    <version>${version.docker.maven.plugin}</version>
                     <configuration>
                         <skip>${test.elasticsearch.run.skip}</skip>
-                        <flavour>${test.elasticsearch.run.flavour}</flavour>
-                        <clusterName>hsearchEsTestCluster</clusterName>
-                        <httpPort>9200</httpPort>
-                        <version>${test.elasticsearch.connection.version}</version>
-                        <timeout>90</timeout><!-- Make it work on our slow CI -->
-                        <!-- auto-create-index is necessary for non-OSS Elasticsearch -->
-                        <autoCreateIndex>true</autoCreateIndex>
-                        <environmentVariables>
-                            <JAVA_HOME>${test.elasticsearch.run.java_home}</JAVA_HOME>
-                        </environmentVariables>
+                        <images>
+                            <image>
+                                <name>docker.elastic.co/elasticsearch/elasticsearch:${test.elasticsearch.connection.version}</name>
+                                <alias>elasticsearch</alias>
+                                <run>
+                                    <env>
+                                        <discovery.type>single-node</discovery.type>
+                                        <!-- fix Docker images for older versions -->
+                                        <xpack.security.enabled>false</xpack.security.enabled>
```

Comment:
Maybe also this, as it was enabled in all of our custom configuration files? Or do you have a reason to avoid this with containers?

```suggestion
                                        <xpack.security.enabled>false</xpack.security.enabled>
                                        <!-- Prevent swapping
                                              This may trigger warnings upon boot if the system is not correctly set up.
      See https://www.elastic.co/guide/en/elasticsearch/reference/7.5/setup-configuration-                                        memory.html#bootstrap-memory_lock
                                         -->
                                        <bootstrap.memory_lock>true</bootstrap.memory_lock>
```